builder/fetch.py

```diff
@@ -113,7 +114,9 @@ def __setitem__(self, item, value):
 
         self.remote = Manifest._fetch_remote()
         self.local = SynchronizedDict(Manifest._load_local())
-        os.makedirs(CACHE_DIR, exist_ok=True)
+        if not os.path.isdir(CACHE_DIR):
+            print('Creating package cache at {}'.format(CACHE_DIR))
+            os.makedirs(CACHE_DIR, exist_ok=True)
```

Comment:
Then what's the point of `exist_ok=True`

Reply:
good point, will fix